src/JuliusSweetland.OptiKey.Core/Models/XmlKeyboardModels/XmlScratchpad.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) 2019 OPTIKEY LTD (UK company number 11854839) - All Rights Reserved
+
+using System.Windows.Controls;
+
+namespace JuliusSweetland.OptiKey.Models
+{
+    /// <summary>
+    /// Interaction logic for XmlScratchpad.xaml
+    /// </summary>
+    public partial class XmlScratchpad : UserControl
+    {
+        public XmlScratchpad()
+        {
+            InitializeComponent();
+        }
+
+        public int Row { get; set; }
```

Comment:
What do the Row and Col properties on these Xml classes do?

Reply:
Apparently nothing. They shall be removed.